src/site/resources/styleguides/google-java-style-20180523/javaguide.html

```diff
@@ -299,6 +308,10 @@ <h4 id="s4.1.2-blocks-k-r-style">4.1.2 Nonempty blocks: K &amp; R style</h4>
     } else {
       lastThing();
     }
+    {
+      int x = foo();
+      frob(x);
+    }
```

Comment:
@Zopsss ,  please add this to Input for this chapter.
to be sure we do not violate such code during whole

Reply:
@Zopsss , please do. Or share link if this already there

Reply:
We've this test case:

https://github.com/checkstyle/checkstyle/blob/db72522f27cf29c67eb85ac62e872812e15d879e/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule412nonemptyblocks/InputRightCurlySwitchCasesBlocks.java#L9-L21

But I think, we should create an issue for this and add more such test cases for different scenarios and also add `// until` comment before it. 

Lemme know if we should do this or not @romani

Reply:
This issue was also detected by cushon from google style guide here: https://github.com/checkstyle/checkstyle/issues/14294#issuecomment-1904385363

The issue was related to indentation problem and incorrect `{` problem.

Reply:
@Zopsss, please put this issue in coverage table de explicit declaration that we have known problems.

Reply:
coverage is update at https://github.com/checkstyle/checkstyle/pull/15417
review thread is resolved.